lib/foreman/renderer/scope/report.rb

```diff
@@ -19,9 +19,14 @@ def initialize(**args)
           desc 'This macro is typically called at the end of the report template, after all rows
             with data has been registered.'
           keyword :format, ReportTemplateFormat.all.map(&:id), desc: 'The desired format of output', default: ReportTemplateFormat.default.id
+          keyword :order, String, desc: "The desired order of the reported data. It needs to be the name of the column or an array of more of them, e.g. <code>'name'</code> or <code>['ip', 'name']</code>.", default: nil
```

Comment:
Should this maybe mention what is the default search order if this parameter is not provided?